reference/websockets/sails.io.js/socket.request.md

```diff
@@ -23,10 +46,10 @@ io.socket.request(options, function (data, jwr)){
 ### Example
 
 ```javascript
-io.socket.request({
+io.socket._request({
```

Comment:
`_request()` is deprecated, see: https://github.com/balderdashy/sails-generate-sails.io.js/blob/master/templates/sails.io.js#L924